src/System.Text.Json/tests/Resources/LargeJson.txt

```diff
@@ -0,0 +1 @@
e native components such as cocoa pods or jars, and look at the massive range of libraries that this framework as available to it to do all manner of UI and application logic things. We\u0027ll even see how to use it on macOS and Windows, including being able to build iOS apps on Windows (with the help of a networked Mac, Apple licensing rules and whatnot).\r\n\r\nWhen looking at naming for this framework, someone suggested Fabulous. By the end of this session you will see why that name stuck.","StartTime":"2019-01-31T16:20:00-08:00","EndTime":"2019-01-31T17:20:00-08:00","Duration":{"Ticks":36000000000,"Days":0,"Hours":1,"Milliseconds":0,"Minutes":0,"Seconds":0,"TotalDays":0.041666666666666664,"TotalHours":1,"TotalMilliseconds":3600000,"TotalMinutes":60,"TotalSeconds":3600},"TrackId":4},{"Track":{"Id":5,"Name":"Room 2"},"Speakers":[],"Id":52,"Title":"Building Clients for OpenID Connect\u002fOAuth 2-based Systems","Abstract":"Using protocols like OpenID Connect and OAuth 2 for authentication and API access can on one hand simply your front-ends dramatically since they don\u2019t have to deal with credentials anymore \u2013 but on the other hand introduces new challenges like choosing the right protocol flow for the given client, secure token storage as well as token lifetime management. This talk gives an overview over the best practices how to solve the above problems for both native server and client-side applications as well as browser-based applications and SPAs.","StartTime":"2019-01-30T13:40:00-08:00","EndTime":"2019-01-30T14:40:00-08:00","Duration":{"Ticks":36000000000,"Days":0,"Hours":1,"Milliseconds":0,"Minutes":0,"Seconds":0,"TotalDays":0.041666666666666664,"TotalHours":1,"TotalMilliseconds":3600000,"TotalMinutes":60,"TotalSeconds":3600},"TrackId":5},{"Track":{"Id":5,"Name":"Room 2"},"Speakers":[],"Id":53,"Title":"Securing Web Applications and APIs with ASP.NET Core 2.2 and 3.0","Abstract":"ASP.NET Core and MVC is a mature and modern platform to build secure web applications and APIs for a while now. Starting with version 2.2, Microsoft makes big investments in the areas of standards-based authentication, single sign-on and API security by including the popular open source project IdentityServer4 in the project templates. This talk gives an overview over the various security features in ASP.NET Core but focuses in particular on the API security scenarios enabled by IdentityServer.","StartTime":"2019-01-30T11:40:00-08:00","EndTime":"2019-01-30T12:40:00-08:00","Duration":{"Ticks":36000000000,"Days":0,"Hours":1,"Milliseconds":0,"Minutes":0,"Seconds":0,"TotalDays":0.041666666666666664,"TotalHours":1,"TotalMilliseconds":3600000,"TotalMinutes":60,"TotalSeconds":3600},"TrackId":5},{"Track":{"Id":5,"Name":"Room 2"},"Speakers":[],"Id":54,"Title":"What you need to know about ASP.NET Core 2.2","Abstract":"Another new version of ASP.NET Core is here and it brings new capabilities, making it easier than ever to build and consume APIs. But there\u0027
```

Comment:
Rather than checking in a 120KB json file, can the test just programmatically generate it?  In that case it'd be able to also do it of varying lengths, sizes, and depths, and we'd end up getting better coverage with less data in the repo.

Reply:
Yes I considered that of course. Only based on time I took this approach short-term. I'll see if I can easily repro the issue with generated data.